### PR TITLE
adding LogicAppResourceId to Action request and response 

### PR DESCRIPTION
**Pull request by ori-licht**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by ori-licht** [comment]
need to make another iteration

**Comment by majastrz** [comment]
>               "$ref": "#/definitions/ActionResponse"

You are effectively adding a required property to a PUT. This will break existing SDKs. These types of changes should be introduced in a new API version.

---
Refers to: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json:413 in 487ed36. [](commit_id = 487ed3629c476e9ddd353f67a80e54fdb737877a, deletion_comment = False)

**Review by majastrz — CHANGES_REQUESTED** [review]
Added a comment about breaking changes. Please take a look.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Comment by ori-licht** [comment]
removed the required

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 1]
```diff
@@ -2589,7 +2589,22 @@
       },
       "type": "object"
     },
+    "ActionPropertiesBase": {
+      "description": "Action property bag base.",
+      "properties": {
+        "logicAppResourceId": {
+          "description": "Logic App Resource Id, providers/Microsoft.Logic/workflows/{WorkflowID}.",
```
Comment (allenjzhang) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: The format description in the description does not seem to be complete given the value shown in the example files.
/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.Logic/workflows/MyAlerts

Please update it to use full value starts with /subscriptions

**Review by allenjzhang — CHANGES_REQUESTED** [review]

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json (hunk 1 above)
Comment (allenjzhang) [reply specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: Everything else looks good. Once updated the description, I will kick off the merge. Thanks

Comment (ori-licht) [reply specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: I've change the description to be consist with the example file

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**